### ISSUE TITLE
data from scheduler isn't being stored correctly

### ISSUE THREAD

**Issue by alxngyn** [osuosl/iam#113]
Scheduler is filling the DB full of wrong data for each table
```
sqlite> select * from disk_size_measurements limit 5;
1||2016-07-06 19:19:07.310998|alembic-java.osuosl.org|0|active
2||2016-07-06 19:19:07.453307|amahi.osuosl.org|0|active
3||2016-07-06 19:19:07.594372|answers.ros.osuosl.org|0|active
4||2016-07-06 19:19:07.736508|apereo.osuosl.org|0|active
5||2016-07-06 19:19:07.878820|app1.osuosl.org|0|active

sqlite> select * from disk_template_measurements limit 5;                      
1||2016-07-06 19:19:57.990596|alembic-java.osuosl.org|disk_template|active
2||2016-07-06 19:19:58.157666|amahi.osuosl.org|disk_template|active
3||2016-07-06 19:19:58.324579|answers.ros.osuosl.org|disk_template|active
4||2016-07-06 19:19:58.466766|apereo.osuosl.org|disk_template|active
5||2016-07-06 19:19:58.642256|app1.osuosl.org|disk_template|active

sqlite> select * from vcpu_count_measurements limit 5;   
1||2016-07-06 19:19:32.184919|alembic-java.osuosl.org|0|active
2||2016-07-06 19:19:32.335290|amahi.osuosl.org|0|active
3||2016-07-06 19:19:32.469617|answers.ros.osuosl.org|0|active
4||2016-07-06 19:19:32.602938|apereo.osuosl.org|0|active
5||2016-07-06 19:19:32.736825|app1.osuosl.org|0|active

```

**Comment by alxngyn** [comment]
fixed by PR #98 
line specifically:
```
collectors.rb

43          @cache.set(node_name, JSON.parse(@template.result(binding)))
```

**Comment by alxngyn** [comment]
pr #98 merged
